gulpfile.js

```diff
@@ -250,15 +252,15 @@ gulp.task('deploy-page', [ 'deploy-data' ], function () {
         });
 
         return extend({}, level, {
-            minSizeProgram: programs.reduce(function (minSizeProgram, program) {
+            minSizeProgram: programs.filter(program => program.legal).reduce(function (minSizeProgram, program) {
```

Comment:
In retrospect, I shouldn't have used `.reduce()` to find something. And there's now `.find()` that can allow both the new filtering + old search to be combined.